test-agents/copilotstudio-webchat-react/build-env.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// This file emulates the process object in node.
+export const process = {
+  env: {
+    // App ID of the App Registration used to log in, this should be in the same tenant as the Copilot.
+    appClientId: 'e42bed64-9aeb-4b3e-9ca9-e9d589737048',
```

Comment:
👆

Reply:
good catch Joel. We dont want to commit files with specific configuration values. 

I'd suggest to grab these from the UI, or at least hardcode in Chat.tsx.